CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ This changelog will be used to generate documentation on [release notes page](ht
 
 ## Version 2.5.0-beta2
 - Remove calculation of sampling-score based on Context.User.Id [For more information, please see issue #625](https://github.com/Microsoft/ApplicationInsights-dotnet/issues/625)
+- New sdk-driven "heartbeat" functionality added which sends health status at pre-configured intervals.
```

Comment:
can you also update readme or create separate file with the instruction on how `Web` package can add properties to the heartbeat?

Reply:
Yes. Will supply this in an upcoming PR (available Monday morning at the latest).